File: python/flask/hello.py
# -*- coding: utf-8 -*-
#
#  hello.py
#  code
#
#  Created by Lars Yencken on 2011-03-28.
#  Copyright 2011 Lars Yencken. All rights reserved.
#

"""
Hello world from Flask micro-framework.
"""

from flask import Flask
app = Flask(__name__)

@app.route('/')
def hello():
    return 'Hello World!'

if __name__ == '__main__':
    app.run()

File: prolog/hello.pl
#!/usr/bin/env swipl -q -g main -s
%
%  hello.pl
%  code
%
%  Created by Lars Yencken on 2011-09-29.
%  Coplright 2011 99designs. All rights reserved.
%

main :- 
    current_prolog_flag(argv, Argv),
    append(_, [--|_Av], Argv), !,
    main(Argv).

main(_Argv) :-
    print('Hello world!\n'),
    halt.
